app/views/drawings/index.html.erb

```diff
@@ -60,6 +60,7 @@
 
       <div class="govuk-grid-column-one-quarter">
         <p style="text-align: right" class="govuk-body">
+          <%= link_to "Edit", edit_planning_application_drawing_path(@planning_application, drawing) %><br />
```

Comment:
Should the reviewer be able to edit this? If not, we should probably reinstate `if current_user.assessor? `